docs/testing/clear-cache.md

```diff
@@ -11,6 +11,9 @@ You can remove an add-in that you've previously sideloaded on Windows, Mac, or i
 
 Additionally, if you make changes to your add-in's manifest (for example, update file names of icons or text of add-in commands), you should clear the Office cache and then re-sideload the add-in using updated manifest. Doing so will allow Office to render the add-in as it's described by the updated manifest.
 
+> [!NOTE]
+> To remove a sideloaded add-in from Excel, OneNote, PowerPoint, or Word on the web, see [Sideload Office Add-ins in Office on the web for testing: Remove a sideloaded add-in](/sideload-office-add-ins-for-testing.md#remove-a-sideloaded-add-in).
```

Comment:
The `/` makes the build point to "docs.microsoft.com/sideload-office-add-ins-for-testing" instead of a relative path.
```suggestion
> To remove a sideloaded add-in from Excel, OneNote, PowerPoint, or Word on the web, see [Sideload Office Add-ins in Office on the web for testing: Remove a sideloaded add-in](sideload-office-add-ins-for-testing.md#remove-a-sideloaded-add-in).
```